web/accounts/models.py

```diff
@@ -71,3 +73,26 @@ def record_consumed_credits(self, credits):
             event_name=settings.STRIPE_METER_EVENT,
             payload={"value": credits, "stripe_customer_id": customer_id},
         )
+
+    def get_credit_usage(self):
+        stripe.api_key = settings.STRIPE_SECRET_KEY
+        customer_id = self.stripe_customer_id
+        if not customer_id:
```

Comment:
we should do this check after the team check since team members have the stripe_customer_id as null